client/components/TestRun/index.jsx

```diff
@@ -967,9 +985,62 @@ const TestRun = () => {
       <div role="complementary">
         <h2 id="test-options-heading">Test Options</h2>
         <ul className="options-wrapper" aria-labelledby="test-options-heading">
-          <li>
+          <li className="raise-issue-container">
```

Comment:
I know this matches a definition in the issue but I can't help but feel like this would make for a challenging user experience, especially on mobile. I almost feel like we want to move this radio into a modal or have a separate link for each command in a future design. For now this works well.

Reply:
That is a good point. I do like the thought of keeping consistency in a single-click approach. The separate link might have some navigational advantages in that it would reduce the distance between the part of the interface with information about a command and the inputs to create the issue.